packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -133,16 +133,23 @@ describe('<ButtonBase />', () => {
       expect(button).not.to.have.attribute('type');
     });
 
-    it('should not use an anchor element if explicit component and href is passed', () => {
```

Comment:
This test case comes from #6882. Hard to tell what the use case. it seems one that we can remove.